release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1619,6 +1619,8 @@ In the following tables, features are marked with the following statuses:
 
 * When running a cluster with FIPS enabled, you might receive the following error when running an older version of the OpenShift CLI (`oc`) on a {op-system-base} 9 system: `FIPS mode is enabled, but the required OpenSSL backend is unavailable`. As a workaround, use the `oc` binary provided with the {product-title} cluster. (link:https://issues.redhat.com/browse/OCPBUGS-23386[*OCPBUGS-23386*])
 
+* In {rh-openstack-first} on {product-version} environments, `IngressController` objects configured with `endpointPublishingStrategy.type: LoadBalancerService` will fail to configure IPv6 correctly. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2263550[*BZ#2263550*], link:https://bugzilla.redhat.com/show_bug.cgi?id=2263552[*BZ#2263552*])
```

Comment:
It's not that it will not configure IPv6 correctly. It will fail on IPv6-first deployments.

Also it's the other way around - OpenShift runs on top of OpenStack. How about:

```
In {product-version} with IPv6 networking running on {rh-openstack-first} environments, `IngressController` objects configured with `endpointPublishingStrategy.type=LoadBalancerService` will not function correctly. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2263550[*BZ#2263550*], link:https://bugzilla.redhat.com/show_bug.cgi?id=2263552[*BZ#2263552*])
```

Reply:
Thanks! I copied it word for word to the release notes :+1: